articles/extensions/authorization-extension/v2/implementation/installation.md

```diff
@@ -88,8 +88,4 @@ Installing this extension creates an `auth0-authz` client for your account. **Do
 
 When you click the link to open the extension for the first time, you will be asked to provide permission for the extension to access your Auth0 account. If you do, you will be redirected to the Authorization Dashboard.
 
-![Authorization Dashboard](/media/articles/extensions/authorization/auth-dashboard-v2.png)
-
-<%= include('./_stepnav', {
- next: ["Configure the Extension", "/extensions/authorization-extension/v2/implementation/setup"]
-}) %>
```

Comment:
Since this custom navigation pointing to the next tutorial was removed, I think it'd be best if you added a next-steps pointing to the "Set up the Extension" step